requirements.txt

```diff
@@ -2,3 +2,4 @@ Django==4.2.6
 djangorestframework==3.14.0
 python-decouple==3.8
 python_jose==3.3.0
+psycopg2==2.9.9
```

Comment:
is this expected?

Reply:
I needed to add that to be able to run it locally